src/liger_kernel/ops/kl_div.py

```diff
@@ -65,16 +66,18 @@ def _kldiv_kernel_forward(
         # KL(y_true || y) = y_true * (log(y_true) - log(y))
         # We compute KL(y_true || y) with y in the log-space
         if not log_target:
-            loss = y_true * (tl.log(y_true) - y)
+            eps = 1e-10
```

Comment:
lets pass eps as a function arg.

Reply:
I suppose we want to propagate it as an argument to the `transformers/LigerKLDiv..` class as well, right?